core/src/main/scala/org/apache/spark/rdd/SqlNewHadoopRDDState.scala

```diff
@@ -20,8 +20,7 @@ package org.apache.spark.rdd
 import org.apache.spark.unsafe.types.UTF8String
 
 /**
- * State for SqlNewHadoopRDD objects. This is split this way because of the package splits.
- * TODO: Move/Combine this with org.apache.spark.sql.datasources.SqlNewHadoopRDD
+ * State for FileScanRDD objects. This is split this way because of the package splits.
  */
 private[spark] object SqlNewHadoopRDDState {
```

Comment:
`SqlNewHadoopRDDState` is definitely not a good name here, how about `InputFileNameHolder`?